README.md

```diff
@@ -1,11 +1,21 @@
 # ACF Content Analysis for Yoast SEO
 WordPress plugin that adds the content of all ACF fields to the Yoast SEO score analysis.
 
-##Description
+## Description
 [Yoast SEO for WordPress](https://yoast.com/wordpress/plugins/) content and SEO analysis does not take in to account the content of a post's [Advanced Custom Fields](http://www.advancedcustomfields.com/). This plugin uses the plugin system of Yoast SEO for WordPress to hook into the analyser in order to add ACF content to the SEO analysis.
 
-This had previously been done by the [WordPress SEO ACF Content Analysis](https://wordpress.org/plugins/wp-seo-acf-content-analysis/) plugin but that no longer works with Yoast 3.0. Kudos to [ryuheixys](https://profiles.wordpress.org/ryuheixys/), the author of that plugin, for the original idea.
+This was previously done by the [WordPress SEO ACF Content Analysis](https://wordpress.org/plugins/wp-seo-acf-content-analysis/) plugin but that no longer works with Yoast 3.0. Kudos to [ryuheixys](https://profiles.wordpress.org/ryuheixys/), the author of that plugin, for the original idea.
 
 This Plugin is compatible with the free ACF 4 Version as well as with the PRO Version 5. Please be aware that it ignores Pro Add-Ons for Version 4. In that case please upgrade to ACF PRO Version 5.
 
 > If you have issues, please [submit them on GitHub](https://github.com/Yoast/yoast-acf-analysis/issues)
+
+
+## Installation
+
+Clone the repo into your plugins folder.
+
+Run:
+* `composer install`
+* `yarn`
+* `yarn build`
```

Comment:
I am not sure if "Installation" is an appropriate subheading for this? If you look at [this repo for woo](https://github.com/Yoast/wpseo-woocommerce#installation) for example, this part is about installing the plugin in WordPress. Maybe we can use something like "Development setup" like in [this repo](https://github.com/Yoast/admin-ui/tree/main/packages/optimize#development-setup)?

Reply:
Agreed. Thank you!